### ISSUE TITLE
 The color contrast ratio for the list items under the card name should be at least 4.5:1

### ISSUE THREAD

**Issue by Kumaraswamy9096** [OfficeDev/office-ui-fabric-react#8634]
<img width="812" alt="Color contrast issue" src="https://user-images.githubusercontent.com/48794043/55778916-d7f73080-5ac1-11e9-824b-ffed400e06ee.PNG">
### Environment Information 
 
- __Package version(s)__: (6.117.0) 
- __Browser and OS versions__: (Microsoft Edge(40.15063.674.0) and 1703)

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/Kumaraswamy9096/pen/JVRBbr?&editable=true
 

 
#### Actual behavior:
 The color contrast between the fore ground and back ground for the list items in the cards is 1.1:1 or below 4.5:1​ in High Contrast White

----Turn on High Contrast White​
----Tab till any list item under the card name using down arrow key​
----Verify whether the color contrast between the foreground and background is 1.1:1 or below 4.5:1​ in High Contrast white
 
#### Expected behavior:
The color contrast between the fore ground and back ground for the list items in the cards should be greater than 4.5:1 ​

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)

**Comment by jdhuntington** [comment]
Hi @Kumaraswamy9096 - thanks for reporting this issue. Would you be willing to add a screenshot to highlight the issue? Afraid I didn't notice the contrast ratio you mention when looking. Thanks!

**Comment by KevinTCoughlin** [comment]
Is the issue here that the _example_ list item contents do not achieve a standard contrast ratio on the production website? It would be great to see the actual validation error text to help triage.
